### PR TITLE
Add explicit job for unknown linux

### PR DESCRIPTION
**Pull request by hasufell**
Depends on https://github.com/haskell/ghcup-metadata/pull/37

Fixes https://github.com/haskell/haskell-language-server/issues/3247

<a href="https://gitpod.io/#https://github.com/haskell/haskell-language-server/pull/3264"></a>

### CODE REVIEW

File: bindist/ghcs [hunk 1]
```diff
@@ -1,8 +1,2 @@
-8.6.5,cabal.project
```
Comment (hasufell) [inline bindist/ghcs:1]: TODO: revert (for testing only)

**Comment by hasufell** [comment]
gitlab CI succeeds (under exclusion of 9.4.x): https://gitlab.haskell.org/maerwald/haskell-language-server/-/jobs/1193161

**Comment by pepeiborra** [comment]
@hasufell do you still need a stamp here? Ping @wz1000

**Comment by hasufell** [comment]
I lost track about this. I believe @wz1000 is taking care of it in another PR.

**Comment by michaelpj** [comment]
This is for the old gitlab CI

**Pull request closed without merging**